### ISSUE TITLE
Formulate learning goals at top of each topic.

### ISSUE THREAD

**Issue by timosachsenberg** [OpenMS/pyopenms-extra#83]
Maybe in a colored box e.g. something like

In this section, you will learn:
- how to load, process and store mass spectra and chromatograms
- how to plot spectra

**Comment by tapaswenipathak** [comment]
What are the remaining documentation todos?

**Comment by jpfeuffer** [comment]
Add learning goals to top of every "applied"/tutorial section of the pyopenms and maybe also the new openms docs.

**Comment by timosachsenberg** [comment]
any update here from last sprint?

**Comment by KyowonJeong** [comment]
Yes. I studied how to put this box and tried a few things. I will try to put this box in each algorithm section by the next week.
